### PR TITLE
Create initial project gov docs

### PR DESCRIPTION
**Pull request by droumis**

### CODE REVIEW

File: doc/governance/project-docs/MEMBERS.md [hunk 1]
```diff
@@ -0,0 +1,14 @@
...
+
+
+The GeoViews Project’s equivalently named documents take precedence over any external materials referenced within this linked document above.
+
+
+| **NAME** | **Role** | **GitHub Handle** |
+| --- | --- | --- |
+| Philipp Rudiger | Project Director | [philippjfr](https://github.com/philippjfr) |
+| Maxime Liquet | Maintainer | [maximlt](https://github.com/maximlt) |
+| Simon Høxbro Hansen | Maintainer | [hoxbro](https://github.com/hoxbro) |
```
Comment (Hoxbro) [inline doc/governance/project-docs/MEMBERS.md:14]:
```suggestion
| Simon Høxbro Hansen | Maintainer | [hoxbro](https://github.com/hoxbro) |
| Andrew Huang | Maintainer | [ahuang11](https://github.com/ahuang11) |
```

**Review by Hoxbro — APPROVED** [review]

**Comment by maximlt** [comment]
+1 for @ahuang11 being added as a new maintainer of GeoViews!

**Review by maximlt — APPROVED** [review]

**Review by philippjfr — APPROVED** [review]
Thanks for adding this!

**Comment by philippjfr** [comment]
One question, should there be a lead maintainer?

**Comment by Hoxbro** [comment]
> One question, should there be a lead maintainer?

Any volunteers?

**Comment by Hoxbro** [comment]
I will merge this to have it in the next release.

**Pull request merged**